src/NzbDrone.Core/Download/Clients/DownloadStation/SharedFolderResolverService.cs

```diff
@@ -0,0 +1,45 @@
+﻿using NLog;
+using NzbDrone.Common.Cache;
+using NzbDrone.Common.Disk;
+using NzbDrone.Core.Configuration;
+using NzbDrone.Core.Download.Clients.DownloadStation.Proxies;
+using NzbDrone.Core.ThingiProvider;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace NzbDrone.Core.Download.Clients.DownloadStation
+{
+    public class SharedFolderResolverService : ISharedFolderResolverService
```

Comment:
Nitpick, but `SharedFolderResolver` is enough, no need to tack `Service` on everything, coz everything is a 'service' anyway.